docs/tools/built-in-tools.md

```diff
@@ -61,13 +61,27 @@ like calculations, data manipulation, or running small scripts.
 
 The GKE Code Executor (`GkeCodeExecutor`) provides a secure and scalable method
 for running LLM-generated code by leveraging the GKE (Google Kubernetes Engine)
-Sandbox environment, which uses gVisor for workload isolation.
-
-For each code execution request, it dynamically creates an ephemeral, sandboxed
-Kubernetes Job with a hardened Pod configuration. This is the recommended
+Sandbox environment, which uses gVisor for workload isolation. It is the recommended
```

Comment:
do not change this language. Also, don't state that something is "recommend" unless the recommendation directly quoting a source.